RestInterface/cafRestUtility.cpp

```diff
@@ -0,0 +1,65 @@
+// ##################################################################################################
+//
+//    Caffa
+//    Copyright (C) 2023- Kontur AS
+//
+//    GNU Lesser General Public License Usage
+//    This library is free software; you can redistribute it and/or modify
+//    it under the terms of the GNU Lesser General Public License as published by
+//    the Free Software Foundation; either version 2.1 of the License, or
+//    (at your option) any later version.
+//
+//    This library is distributed in the hope that it will be useful, but WITHOUT ANY
+//    WARRANTY; without even the implied warranty of MERCHANTABILITY or
+//    FITNESS FOR A PARTICULAR PURPOSE.
+//
+//    See the GNU Lesser General Public License at <<http://www.gnu.org/licenses/lgpl-2.1.html>>
+//    for more details.
+//
+// ##################################################################################################
+
+#include "cafRestUtility.h"
+
+#include <boost/algorithm/string.hpp>
+#include <boost/archive/iterators/base64_from_binary.hpp>
+#include <boost/archive/iterators/binary_from_base64.hpp>
+#include <boost/archive/iterators/remove_whitespace.hpp>
+#include <boost/archive/iterators/transform_width.hpp>
+
+namespace caffa::rpc
+{
+
+std::string decodeBase64( std::string input )
```

Comment:
Maybe input should be const reference here?

Reply:
That would mean having to copy it immediately after, because the code modifies it. I think the code looks simpler when just passing by copy.